counterpartylib/test/integration_test.py

```diff
@@ -5,16 +5,18 @@
 from counterpartylib.test import util_test
 
 
-def test_scenario(scenario_name, base_scenario_name, transactions, rawtransactions_db):
+def test_scenario(scenario_name, base_scenario_name, transactions):
```

Comment:
maybe I'm missing something, but I don't see an updated call to this function anywhere in the diff set (was trying to trace through the differences with `--savescenarios`)